data/templates/css/wikimedia.scss

```diff
@@ -566,14 +566,27 @@ $spacing-elements: 30px;
 @media (max-width: 360px) {
     #bodycontents {
         .eos-article-title h1 {
-            font: 700 1.4rem $font-secondary;
-            margin-top: 0px;
-            padding: 0px 0px 5px 15px;
-            text-align: left;
+            display: none;
         }
     }
     html {
         -webkit-text-size-adjust:none;
         font-size:100%
-}
+    }
+    .mw-parser-output{
+        display: flex;
+        flex-direction: column;
+    }
+
+    .mw-parser-output > *{
+        order: 2;
+    }
+
+    .mw-parser-output > p:first-of-type{
+        order: 0;
+    }
+
+    .infobox {
+        order: 1;
+    }
```

Comment:
Neat, I didn't know about this property. That's handy (https://css-tricks.com/snippets/css/a-guide-to-flexbox/)